infra/resources/prod/servicebus_topic.tf

```diff
@@ -0,0 +1,101 @@
+/////////////////////////
+//      TOPICS         //
+/////////////////////////
+resource "azurerm_servicebus_topic" "io_auth_topic" {
+  name         = "io-auth-topic"
```

Comment:
I suggest to use a more meaningful name, we'll use this topic for session events

Reply:
ok! let's rename the resource, too